### ISSUE TITLE
Fails to compare Sets

### ISSUE THREAD

**Issue by sean256** [angus-c/just#217]
```
const a = { name: 'foo', someSet: new Set([1, 2]) };
const b = { name: 'foo', someSet: new Set([1, 2, 4]) };

console.log(compare(a, b)); // outputs true despite `.someSet` having different values
```

**Comment by angus-c** [comment]
good call, will take a look when I get a chance

**Comment by angus-c** [comment]
Fixed in just-compare@1.4.0
